var/spack/repos/builtin/packages/mvapich2-gdr/package.py

```diff
@@ -0,0 +1,198 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os.path
+import sys
+
+
+class Mvapich2Gdr(AutotoolsPackage):
+    """MVAPICH2-GDR is an optimized version of the MVAPICH2 MPI library for
+    GPU-enabled HPC and Deep Learning Applications. MVAPICH2-GDR is not
+    installable from source and is only available through a binary mirror.
+    If you do not find the binary you're looking for, send us an email at
+    mvapich@cse.ohio-state.edu. The binary mirror url is:
+    http://mvapich.cse.ohio-state.edu:8080/download/mvapich/spack-mirror/mvapich2-gdr/
+    """
+
+    homepage = 'http://mvapich.cse.ohio-state.edu'
+    url      = ('http://mvapich.cse.ohio-state.edu:8080/download/mvapich/'
+                'spack-mirror/mvapich2-gdr/mvapich2-gdr-2.3.4.tar.gz')
```

Comment:
You should keep this on a single line, Spack exempts URLs from line-length checks.

Reply:
Thanks for the feedback. Just made the requested change.